conferences/src/App.js

```diff
@@ -9,6 +9,8 @@ import { ThemeProvider, CssBaseline } from '@mui/material';
 import createEmotionCache from './util/createEmotionCache';
 import lightTheme from './styles/theme/lightTheme';
 
+import Header from './components/Header';
+
 import awsexports from './aws-exports';
```

Comment:
This file is missing. Can’t run the thing because of it.

Reply:
`aws-exports.js` is not usually stored in the repo